backend/app/middlewares/base_decoraters.py

```
@@ -1,23 +1,100 @@
+                    inspect.Parameter.POSITIONAL_OR_KEYWORD, 
+                    annotation=Request
+                )
+                visible_params.insert(0, request_param)
+            
+            # Update the wrapper's signature
+            wrapper.__signature__ = inspect.Signature(
+                parameters=visible_params, 
+                return_annotation=original_sig.return_annotation
+            )
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)